src/pages/Withdraw/index.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Layout } from '../../components/ui/Layout';
+import { Input } from '../../components/ui/Input';
+import { Button } from '../../components/ui/Button';
+import { Card as CardSvg } from '../../components/ui/Icons';
+import { Card } from '../../components/card';
+import { Label } from '../../components/ui/Label';
+import { Text } from '../../components/ui/Text';
+import { Title } from '../../components/ui/Title';
+
+import style from './styles.module.scss';
+
+export const Withdraw = () => {
+  const { t, i18n } = useTranslation();
+  const [sum, setSum] = useState(50);
+  const [email, setEmail] = useState('nicholas.angel@gmail.com');
+
+  const data = {
+    max: 84,
+    currency: 'AED',
+    card: {
+      number: '11112222333344443456',
+      date: '10/2024',
+      name: 'NICHOLAS ANGEL',
+    },
+  };
+
+  const numberFormat = new Intl.NumberFormat(i18n.language, {
+    useGrouping: false,
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 2,
+  });
+
+  const currencyFormat = new Intl.NumberFormat(i18n.language, {
+    style: 'currency',
+    currency: data.currency,
+    useGrouping: false,
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 2,
+  });
+
+  const renderCurrency = (value) => {
+    return currencyFormat.format(value).replace(numberFormat.format(value), '').replace(' ', '');
+  };
+
+  const onChangeSum = (e) => {
+    const value = e.currentTarget.value;
+    let result = sum;
+
+    if (value[0] === '.' || value[0] === ',') {
+      result = `0${value}`.replace(',', '.');
+    } else if (/^[\d]*(\.|\,)?[\d]{0,2}$/.test(value)) {
```

Comment:
Better change it on send to disturb a customer less

Reply:
"Intl.NumberFormat" only works through the point

Reply:
I will try to get around

Reply:
I mean, it's confusing if a value changes on input, but we may replace chars right before the value goes to a server

Reply:
In such case it's OK, I think